SpatialGDK/Source/SpatialGDK/Private/Tests/TestDefinitions.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Misc/AutomationTest.h"
```

Comment:
I'm used to seeing a distinction made between `#include ""` and `#include <>`. When the file exists outside the module I would prefer the <> variant since this will make the compiler search outside the module first.

Reply:
Updated includes.